dedupe/convenience.py

```diff
@@ -88,101 +88,101 @@ def randomPairsWithReplacement(n_records: int, sample_size: int) -> IndicesItera
     return ((p.item(), q.item()) for p, q in random_indices)
 
 
+def _print(*args) -> None:
+    print(*args, file=sys.stderr)
+
+
+LabeledPair = Tuple[TrainingExample, Literal["match", "distinct"]]
+
+
+def _mark_pair(deduper: dedupe.api.ActiveMatching, labeled_pair: LabeledPair) -> None:
+    record_pair, label = labeled_pair
+    examples: TrainingData = {"distinct": [], "match": []}
+    examples[label].append(record_pair)
+    deduper.mark_pairs(examples)
+
+
 def console_label(deduper: dedupe.api.ActiveMatching) -> None:  # pragma: no cover
     '''
-   Train a matcher instance (Dedupe, RecordLink, or Gazetteer) from the command line.
-   Example
+    Train a matcher instance (Dedupe, RecordLink, or Gazetteer) from the command line.
+    Example
 
-   .. code:: python
+    .. code:: python
 
-      > deduper = dedupe.Dedupe(variables)
-      > deduper.prepare_training(data)
-      > dedupe.console_label(deduper)
+       > deduper = dedupe.Dedupe(variables)
+       > deduper.prepare_training(data)
+       > dedupe.console_label(deduper)
     '''
 
     finished = False
     use_previous = False
-    fields = unique(field.field
-                    for field
-                    in deduper.data_model.primary_fields)
+    fields = unique(field.field for field in deduper.data_model.primary_fields)
 
     buffer_len = 1  # Max number of previous operations
-    examples_buffer: List[Tuple[TrainingExample, Literal['match', 'distinct', 'uncertain']]] = []
-    uncertain_pairs: List[TrainingExample] = []
+    unlabeled: List[TrainingExample] = []
+    labeled: List[LabeledPair] = []
 
     while not finished:
         if use_previous:
-            record_pair, _ = examples_buffer.pop(0)
+            record_pair, _ = labeled.pop(0)
```

Comment:
we would need to pop twice in the case of unsure, right?

Reply:
Oops, yeah. See fixup commit